front/app/modules/commercial/user_custom_fields/utils/data/rename.ts

```diff
@@ -0,0 +1,23 @@
+type RenameInstructions<T> = {
+  [Poperty in keyof T]?: string;
+};
+
+const rename = <T>(
```

Comment:
Not 100% clear to me what is this renaming until I look at the code. :)

Reply:
A 'column' if you see this Array of Objects as a representation of a table. Again inspired by libraries such as [tidyverse](https://dplyr.tidyverse.org/reference/rename.html)